libraries/messages/src/mav_message.rs

```diff
@@ -0,0 +1,20 @@
+use mavlink;
+use heapless::Vec;
+
+pub const MAV_HEADER_MAIN: mavlink::MavHeader = mavlink::MavHeader {
+    system_id: 1,
+    component_id: 1,
+    sequence: 42,
```

Comment:
Yeah, I would. While I don't think we would ever ask for a retransmission (at least not at the MAVLink level), this would allow the ground station to keep track of the number of dropped packets.